.github/workflows/lock.yml

```diff
@@ -11,9 +11,9 @@ jobs:
       - uses: dessant/lock-threads@v3
         with:
           github-token: ${{ github.token }}
```

Comment:
This can be removed as well

Reply:
true, according to doc, all what we need is `issue-inactive-days` and `pr-inactive-days` as for rest of inputs we are using defaults, tho i'm not sure if we want to clean this up

Reply:
i'm happy to leave it. no harm in having it!